web/models/organization_invite.ex

```diff
@@ -0,0 +1,54 @@
+defmodule CodeCorps.OrganizationInvite do
+  @moduledoc """
+  Handles inviting organizations via email
+  """
+
+  use CodeCorps.Web, :model
+
+  schema "organization_invites" do
+    field :code, :string
+    field :email, :string
+    field :title, :string
+    field :fulfilled, :boolean, default: false
+
+    timestamps()
+  end
+
+  @doc """
+  Builds a changeset based on the `struct` and `params`.
+  """
+  def changeset(struct, params \\ %{}) do
+    struct
+    |> cast(params, [:email, :title, :fulfilled])
+    |> validate_required([:email, :title])
+    |> validate_format(:email, ~r/@/)
+    |> validate_change(:fulfilled, &check_fulfilled_changes_to_true/2)
+  end
+
+  @doc """
+  Builds a changeset for creating an organization invite.
+  """
+  def create_changeset(struct, params) do
+    struct
+    |> changeset(params)
+    |> generate_code
+  end
+
+  defp generate_code(changeset) do
+    case changeset do
+      %Ecto.Changeset{valid?: true}  ->
+        length = 10
+        code = :crypto.strong_rand_bytes(length) |> Base.encode64 |> binary_part(0, length)
```

Comment:
The issue mentioned a short key. I took the short length to be 10. 
Also I assumed the `:crypto.strong_rand_bytes(length)` would be random enough that collisions won't occur with other `code` values in the database. Field `code` has a unique index on it.  In case it needs to checked, would it be better  to keep generating code until a unique string is found, that does not clash with the database value of field?